Procfile

```diff
@@ -1,10 +1,10 @@
 build: while true; do make -qs bin/goalert || make bin/goalert || (echo '\033[0;31mBuild Failure'; sleep 3); sleep 0.1; done
 
 @watch-file=./bin/goalert
-goalert: ./bin/goalert -l=localhost:3030 --ui-url=http://localhost:3035 --db-url=postgres://goalert@localhost:5432/goalert?sslmode=disable --listen-sysapi=localhost:1234 --listen-prometheus=localhost:2112
+goalert: ./bin/goalert -l=localhost:3030 --ui-dir=web/src/build --db-url=postgres://goalert@localhost:5432/goalert?sslmode=disable --listen-sysapi=localhost:1234 --listen-prometheus=localhost:2112
 
 smtp: go run github.com/mailhog/MailHog -ui-bind-addr=localhost:8025 -api-bind-addr=localhost:8025 -smtp-bind-addr=localhost:1025 | grep -v KEEPALIVE
 prom: bin/tools/prometheus --log.level=warn --config.file=devtools/prometheus/prometheus.yml --storage.tsdb.path=bin/prom-data/ --web.listen-address=localhost:9090
 
 @watch-file=./web/src/webpack.config.js
-ui: yarn workspace goalert-web webpack serve --config ./webpack.config.js
+ui: yarn workspace goalert-web webpack --config ./webpack.config.js --watch
```

Comment:
I agree with ignoring `node_modules` -- I would have assumed that was the default since `node_modules` can be so heavy.

As for the flag -- I think that may be a little surprising and would prefer to keep it explicit. `webpack --config ./file-name.js` will build and exit with prod and cypress config files. Rather than having dev behave differently, I think having it called out in the command is clearer.

That being said there's a good chance esbuild will be replacing this soon anyhow, so it's not too impactful.